meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor

```diff
@@ -133,6 +133,7 @@ fi
 # Try to stop old supervisor to prevent it deleting the intermediate images while downloading the new one
 echo "Stop supervisor..."
 systemctl stop resin-supervisor
+$DOCKER rm --force resin_supervisor
```

Comment:
Done, but I also moved the supervisor.conf change so that it is only changed after a successful pull